apps/next-evans/components/core/BackgroundImage.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Image from 'next/image';
+import cx from 'classnames';
+
+interface Props {
+  src: string;
+  fit: 'cover' | 'contain' | 'fill' | 'none' | 'scale-down';
+  position:
+    | 'top'
+    | 'bottom'
+    | 'center'
+    | 'left'
+    | 'right'
+    | 'top-left'
+    | 'top-right'
+    | 'bottom-left'
+    | 'bottom-right';
+  children?: React.ReactNode;
+  className?: string;
+}
+
+const fitMap = {
+  cover: `object-cover`,
+  contain: `object-contain`,
+  fill: `object-fill`,
+  none: `object-none`,
+  'scale-down': `object-scale-down`,
+};
+
+const positionMap = {
+  top: `object-top`,
+  bottom: `object-bottom`,
+  center: `object-center`,
+  left: `object-left`,
+  right: `object-right`,
+  'top-left': `object-left-top`,
+  'top-right': `object-right-top`,
+  'bottom-left': `object-left-bottom`,
+  'bottom-right': `object-right-bottom`,
+};
+
+const BackgroundImage: React.FC<Props> = ({
+  src,
+  className,
+  children,
+  fit,
+  position,
+}) => (
+  <div className={cx(`relative`, className)}>
+    <Image
+      src={src}
+      width={1000}
+      height={1000}
+      alt=""
```

Comment:
let's also add `role="presentation"` here for accessibility. i think the empty alt is enough, but it won't hurt